imantics/image.py

```diff
@@ -65,10 +65,11 @@ def from_coco(cls, coco, dataset=None):
             'date_captured': coco.get('date_captured')
         })
 
+
         data = {
             'id': coco.get('id', 0),
-            'width': coco.get('width', 0),
-            'height': coco.get('height', 0),
+            'width': coco.get('height', 0),
```

Comment:
Why are these being inverted?

Reply:
old test that I've forgot to remove sorry